### DISCUSSION TITLE
Modify real response from server

### DISCUSSION THREAD

**Discussion by Luperco** [mockoon/mockoon#1077] [Help]
Hi all,
is it possible, with Mockoon, mock a route so that it is forwarded to server and add or modify some data of the server response?

I need to test a new functonality with my frontend. The api is changed in a way that some data is added to an existing resource.
I would use the old api call (while they develop new functionality) adding the new data but without mocking the entire call that is quite complex.

Thanks.

**Comment by 255kb** [comment]
Hi @Luperco 

This is not possible aside from [adding headers to the proxied response](https://mockoon.com/docs/latest/proxy-mode/#proxy-headers). 
It's not something that we have on the roadmap or that has been requested before.
